src/pages/bridge/TransferDialog.tsx

```diff
@@ -11,12 +14,10 @@ import { convertRawHexKeyToPublicKey } from '../../helpers/stellar';
 import { useGlobalState } from '../../GlobalStateProvider';
 import { useSecurityPallet } from '../../hooks/spacewalk/security';
 import { calculateDeadline, currencyToString, deriveShortenedRequestId } from '../../helpers/spacewalk';
-import { useEffect, useMemo, useState } from 'react';
-import { DateTime } from 'luxon';
 import CancelledDialogIcon from '../../assets/dialog-status-cancelled';
 import WarningDialogIcon from '../../assets/dialog-status-warning';
 import TransferCountdown from '../../components/TransferCountdown';
-import { hexToU8a } from '@polkadot/util';
+import { useVaultRegistryPallet } from '../../hooks/spacewalk/vaultRegistry';
```

Comment:
See my comment below
```suggestion
import { useVaultRegistryPallet } from '../../hooks/spacewalk/vaultRegistry';
import { toTitle } from '../../helpers/string';
```